src/site/layouts/landing_page.drupal.liquid

```diff
@@ -110,14 +110,14 @@
           </li>
         </ul>
 
+        {% if fieldLinks != empty %}
```

Comment:
When I try this in dev, fieldLinks is an empty array, which passes this check. So I still see the box in the sidebar. I think you want to check that it's not empty and that the length is greater than 0.

Reply:
Good catch, @jbalboni - updated